Blueprints/BlueprintDefinitions/vs2022/EmptyServerless/template/src/BlueprintBaseName.1/Readme.md

```diff
@@ -44,4 +45,77 @@ Deploy application
 
 Arm64 support is provided by the AWS Graviton2 processor. For many Lambda workloads Graviton2 provides the best price performance.
 
-If you want to run your Lambda on a Graviton2 Arm64 processor, all you need to do is replace `x86_64` with `arm64` under `"Architectures": ` in the `serverless.template` file. Then deploy as described above. 
\ No newline at end of file
+If you want to run your Lambda on a Graviton2 Arm64 processor, all you need to do is replace `x86_64` with `arm64` under `"Architectures": ` in the `serverless.template` file. Then deploy as described above. 
+
+## Lambda Annotations
+This template now uses [Lambda Annotations](https://github.com/aws/aws-lambda-dotnet/blob/master/Libraries/src/Amazon.Lambda.Annotations/README.md) to bridge the gap between the Lambda programming model and a more idiomatic .NET model.
+
+This automatically handles reading parameters from an APIGatewayProxyRequest and returning an APIGatewayProxyResponse. 
+
+It also generates the function resources in a JSON or YAML CloudFormation template based on your function definitions, and keeps them updated.
+
+### Using Annotations without API Gateway
+If you still wish to use Lambda Annotations for the CloudFormation template syncing without taking advantage of the integration with API Gateway, replace your function definition with the following:
+```
+    [LambdaFunction(PackageType = LambdaPackageType.Zip, Policies = "AWSLambdaBasicExecutionRole", MemorySize = 256, Timeout = 30)]
```

Comment:
Same comment as the image based version use a non API Gateway looking example.

Reply:
Done, I did swap in `AWSLambdaSQSQueueExecutionRole` for the policy in the SQS example